common/app/model/content.scala

```diff
@@ -75,7 +75,7 @@ final case class Content(
   lazy val shortUrlId = fields.shortUrlId
   lazy val shortUrlPath = shortUrlId
   lazy val discussionId = Some(shortUrlId)
-  lazy val isGallery = metadata.contentType == DotcomContentType.Gallery
+  lazy val isGallery = metadata.contentType.exists(c => c == DotcomContentType.Gallery)
```

Comment:
could do (_ == DotcomContentType.Gallery)

Reply:
`contains(DotcomContentType.Gallery)` would be the idiomatic scala way